remapping_pipeline.nf

```diff
@@ -0,0 +1,350 @@
+#!/usr/bin/env nextflow
+
+def helpMessage() {
+    log.info"""
+    Index the provided genome with bowtie and samtools
+    Usage:
+            --oldgenome                     Fasta file containing the reference genome.
+            --outdir                    Directory where the index will be deployed.
+    """
+}
+params.flankingseq = 50
+params.scorecutoff = 0.6
+params.diffcutoff =  0.04
+params.outfile = "$baseDir/resources/remap_vcf.vcf"
+
+// Show help message
+if (params.help) exit 0, helpMessage()
```

Comment:
This two-line block (Show help message comment + command) should probably be moved to directly after the definition of `helpMessage`